src/basic/in-addr-util.h

```diff
@@ -30,6 +30,8 @@ int in_addr_is_link_local(int family, const union in_addr_union *u);
 bool in4_addr_is_localhost(const struct in_addr *a);
 int in_addr_is_localhost(int family, const union in_addr_union *u);
 
+bool in4_addr_is_suitable_for_dnsntp(const struct in_addr *a);
```

Comment:
this functions sounds awfully specific for in-addr-util.h in basic/. Maybe move to network-internal.h instead?

Reply:
or alternatively, maybe call this `in4_addr_is_non_local()` and leave it in basic/? After all, that's what this actually checks. by renaming it it would become generic enough to stay in in-addr-util.h I figure...

Reply:
I like `in4_addr_is_non_local()` the best... will change back.